### PR TITLE
Fix issues reported in the console

### PR DESCRIPTION
**Pull request by DonOmalVindula**
### Purpose
$subject

### Related Issues
- Issue `#1` or (None)

### Related PRs
- Related PR `#1` or (None)

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/users/pages/users.tsx [hunk 1]
```diff
@@ -132,6 +132,75 @@ const UsersPage: FunctionComponent<UsersPageInterface> = (
...
 
+    /**
+     * Fetch the list of available userstores.
+     */
+    useEffect(() => {
+        if (!OrganizationUtils.isCurrentOrganizationRoot()) {
+            return;
+        }
+
+        if(CommonHelpers.lookupKey(tenantSettings, username) !== null) {
```
Comment (brionmario) [inline apps/console/src/features/users/pages/users.tsx:143]:
```suggestion
if (CommonHelpers.lookupKey(tenantSettings, username) !== null) {
```

File: apps/console/src/features/users/pages/users.tsx [hunk 2]
```diff
@@ -132,6 +132,75 @@ const UsersPage: FunctionComponent<UsersPageInterface> = (
...
+        if (init.current) {
+            init.current = false;
+        } else {
+            if (emailVerificationEnabled !== undefined) {
+                setShowWizard(true);
+            }
+        }
+    }, [ emailVerificationEnabled ]);
+
+    const getReadOnlyUserStoresList = () => {
```
Comment (brionmario) [inline apps/console/src/features/users/pages/users.tsx:198]:
```suggestion
    const getReadOnlyUserStoresList = (): void => {
```

**Review by brionmario — APPROVED** [review]

File: apps/console/src/features/users/pages/users.tsx (hunk 1 above)
Comment (DonOmalVindula) [reply apps/console/src/features/users/pages/users.tsx]: Fixed with 4a386ebbe356947490b8274a2d2c325000665799

File: apps/console/src/features/users/pages/users.tsx (hunk 2 above)
Comment (DonOmalVindula) [reply apps/console/src/features/users/pages/users.tsx]: Fixed with 4a386ebbe356947490b8274a2d2c325000665799

**Review by brionmario — APPROVED** [review]

**Pull request merged**